docs/source/user_devices.rst

```diff
@@ -13,4 +13,12 @@ This import statement assumes your custom device follows the new device structur
 
 Note that both the `userlib` path and the `user_devices` folder name can be custom configured in the `labconfig.ini` file. The `user_devices` folder must be in the `userlib` path. If a different `user_devices` folder name is used, the import uses that folder name in place of `user_devices` in the above import statement.
 
-Note that we highly encourage everyone that adds support for new hardware to consider making a pull request to **labscript-devices** so that it may be added to the mainline and more easily used by other groups.
\ No newline at end of file
+Note that we highly encourage everyone that adds support for new hardware to consider making a pull request to **labscript-devices** so that it may be added to the mainline and more easily used by other groups.
+
+3rd Party Devices
+-----------------
+
+Below is a list of 3rd party devices developed by users of **labscript** that can be used via the `user_devices` mechanism described above. If you would like to add your repository to this list, :doc:`please contact us or make a pull request<labscript-suite:contributing>`.
```

Comment:
Maybe add something along the lines of "These repositories are not tested or maintained by the **labscript** development team. They may have some lab-specific implementation that may not work for your apparatus, though even then the code may still be a good reference. Direct any questions about these repositories to their owner." Feel free to edit or reword any of that of course

Reply:
Well noted! I've gone ahead and added the disclaimer (and updated your affiliation). Unfortunately I have to be a little cagey about my affiliation, because _reasons_, so hopefully people don't mind.

Reply:
Haha no worries, I think that's totally reasonable

And the updated text looks great!